### PR TITLE
Feature/about page

### PR DESCRIPTION
**Pull request by richardverheyen**
I basically just made the about mission-values about page using some of the tricks you used in Finsec. You might want to just ignore the about-nav stuff, I made a little stylesheet for that too. DO what you will

### CODE REVIEW

File: src/styles/_about-nav.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#about-nav {
```
Comment (janwerkhoven) [inline src/styles/_about-nav.scss:1]: Hehe you did everything like I would 👍

**Comment by janwerkhoven** [comment]
Looking good, you did many things like I would. I'll merge in manually.

File: src/styles/_about-nav.scss (hunk 1 above)
Comment (richardverheyen) [reply src/styles/_about-nav.scss:1]: Awwww yeah boi

**Comment by janwerkhoven** [comment]
Merged in manually

**Pull request closed without merging**